src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -201,74 +179,45 @@ public void testNA12878NormalNormalFiltering() throws Exception {
 
         final File filteredVcf = createTempFile("filtered", ".vcf");
 
-        // run FilterMutectCalls
-        new Main().instanceMain(makeCommandLineArgs(Arrays.asList(
-                "-R", b37Reference,
-                "-V", unfilteredVcf.getAbsolutePath(), "-O", filteredVcf.getAbsolutePath(),
-                "--" + M2FiltersArgumentCollection.TUMOR_SEGMENTATION_LONG_NAME, segments.getAbsolutePath(),
-                "--" + M2FiltersArgumentCollection.CONTAMINATION_TABLE_LONG_NAME, contamination.getAbsolutePath(),
-                "--" + FilterMutectCalls.FILTERING_STATS_LONG_NAME, stats.getAbsolutePath()),
-                FilterMutectCalls.class.getSimpleName()));
+        runFilterMutectCalls(unfilteredVcf, filteredVcf, b37Reference,
+                args -> args.addFileArgument(M2FiltersArgumentCollection.TUMOR_SEGMENTATION_LONG_NAME, segments),
+                args -> args.addFileArgument(M2FiltersArgumentCollection.CONTAMINATION_TABLE_LONG_NAME, contamination),
+                args -> args.addFileArgument(FilterMutectCalls.FILTERING_STATS_LONG_NAME, stats));
```

Comment:
@davidbenjamin Totally random request: while you're in here, can you change this test so it doesn't write to the checked-in version of this file (running the test always leaves the working directory in a modified state).

Reply:
@cmnbroad Hmm, I bet that file shouldn't be checked in at all.  Let me fix that.

Reply:
Thanks @davidbenjamin .